sp/include/optee_sp_internal_api.h

```diff
@@ -0,0 +1,67 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2020-2021, Arm Limited.
+ */
+
+#ifndef SP_INCLUDE_OPTEE_SP_INTERNAL_API_H_
+#define SP_INCLUDE_OPTEE_SP_INTERNAL_API_H_
+
+#include <stddef.h>
+#include <stdint.h>
+#include "compiler.h"
+#include "util.h"
+
+/*
+ * The file describes the API between the SP dev kit and the SP. It contains the
+ * function prototypes that the user code must define.
+ * The SP code base should also contain a header file named
+ * "optee_sp_user_defines.h" for passing the following definitions to the SP dev
+ * kit:
+ * * OPTEE_SP_HEAP_SIZE: Heap size in bytes
+ * * OPTEE_SP_UUID: UUID of the SP as an sp_uuid structure
+ * * OPTEE_SP_STACK_SIZE: Stack size in bytes
+ * * OPTEE_SP_FLAGS: SP attributes (currently none available, set to zero)
```

Comment:
Do SPs plan to have attributes?

Reply:
The [same](https://github.com/OP-TEE/optee_os/pull/4732#discussion_r691247152) goes to the `flags` field but I agree that `OPTEE_SP_FLAGS` is useless as it can be replaced by setting the field to 0 directly in `sp_header.c`.